src/collective/mustread/models.py

```diff
@@ -17,10 +17,12 @@ class MustRead(Base):
     userid = Column(Unicode(255), nullable=True)
     read_at = Column(DateTime, nullable=True)
     deadline = Column(DateTime, nullable=True)
+    scheduled_at = Column(DateTime, nullable=True)
+    scheduled_by = Column(String(255))
```

Comment:
This is very implicit, since it's not part of the interface contract of `schedule_must_read`. Which means you're implicitly adding the `request` to the interface contract of a backend component which is request-agnostic. In other words this column implies you're turning a database utility into a browser view. This will make testing and debugging much more difficult. If you need `scheduled_by`, add it to the contract of `schedule_must_read`.